src/System.Runtime.Extensions/ref/System.Runtime.Extensions.cs

```diff
@@ -975,6 +991,14 @@ public static partial class WebUtility
         public static byte[] UrlEncodeToBytes(byte[] value, int offset, int count) { throw null; }
     }
 }
+namespace  System.Reflection
+{
+    public class AssemblyNameProxy : System.MarshalByRefObject
+    {
+        public AssemblyNameProxy() { }
+        public System.Reflection.AssemblyName GetAssemblyName(System.String assemblyFile) { return default(System.Reflection.AssemblyName); }
```

Comment:
It did used to be ```return default(T);```, but we've relatively recently switched all of the contracts both in corefx and in standard to use ```throw null;``  It's easier to read, leads to more consistency, and also results in less IL (especially when there are out parameters) / smaller contract assemblies.